client/src/js/components/bhGenderSelect/bhGenderSelect.js

```diff
@@ -0,0 +1,31 @@
+angular.module('bhima.components')
+  .component('bhGenderSelect', {
+    templateUrl : 'js/components/bhGenderSelect/bhGenderSelect.tmpl.html',
+    controller  : GenderSelectController,
+    bindings    : {
+      value            : '<',
+      onSelectCallback : '&',
+      required         : '<?',
+      validateTrigger  : '<?',
```

Comment:
Maybe if we added an `includeOther` flag, we would be able to use this in more places?